config/webpack/webpack.common.js

```diff
@@ -19,6 +19,7 @@ const includeModules = [
     'react-native-modal',
     'react-native-onyx',
     'react-native-gesture-handler',
+    'toggle-switch-react-native',
```

Comment:
Since you built a switch from scratch, don't forget to remove this unused dependency

Reply:
good catch!